Src/Interface/React/TopbarBroadcast/init.lua

```diff
@@ -108,22 +108,22 @@ local function TopbarBroadcast(properties: {
 		BackgroundTransparency = 1,
 
 		Size = UDim2.new(1, 0, 0, insetSizeY - topbarPositionPadding),
-		Position = UDim2.new(0, 0, 0, topbarPositionPadding)
+		Position = UDim2.new(0, 0, 0, topbarPositionPadding),
 	}, {
 		backgroundFrame = React.createElement("Frame", {
 			Size = inAnimationHook.progress:map(function(value)
-				return UDim2.new(0, backgroundSizeX * value, 1  * math.min(value + 0.75, 1), 0) 
+				return UDim2.new(0, backgroundSizeX * value, 1 * math.min(value + 0.75, 1), 0)
 			end),
 
 			AnchorPoint = Vector2.new(0.5, 0.5),
 			Position = UDim2.fromScale(0.5, 0.5),
 			BackgroundColor3 = InterfaceTheme.Secondary.Black,
 			BackgroundTransparency = inAnimationHook.progress:map(function(value)
 				return 1 - (0.7 * value)
-			end)
+			end),
 		}, {
-			UICorner = React.createElement("UICorner", {
-				CornerRadius = UDim.new(isOldTopbar and 0 or 1, InterfaceTheme.Padding)
+			uiCorner = React.createElement("UICorner", {
+				CornerRadius = UDim.new(isOldTopbar and 0 or 1, InterfaceTheme.Padding), -- this one is still padding because i think its made to match roblox
```

Comment:
then we should isolate this entirely so it doesn't depend on this variable! whoops.. :eyes: